templates/components/badgeInstance.jsx

```diff
@@ -0,0 +1,24 @@
+var React = require('react');
+
+
+var BadgeInstance = React.createClass({
+  render: function() {
+    var badge = this.props.badge,
+        doiRe = /(10\.\d{3}\d+)\/(.*)\b/,
+        m = doiRe.exec(badge.evidenceUrl),
+        paperUrl = '/v/#/papers/' + m[1] + '/' + encodeURIComponent(m[2]) + '/badges';
+
+    return (
+      <div className="badge pure-u-1-4">
+          <div className="pure-u-1-2">
+            <img src={ badge.badge.imageUrl }/>
```

Comment:
hey! I updated the variable name to 'badgeInstance'  -- in the open badges schema, a badgeInstance has a badge. 

So now, it's badgeInstance.badge instead of badge.badge...

We still need badgeInstance to get things like badgeInstance.orcid